src/deriver/api.py

```diff
@@ -357,6 +362,90 @@ def scan_selfies(self):
 
         return good_children, all_filtered_children
 
+    def derive_smiles_gb(self, n_children: int = 100, mut_rate: float = 0.01):
+        children = []
+        good_children = []
+
+        if self.data.filter:
+            filter_params = self.data.filter_params
+        else:
+            logger.warning("Warning: No filter has been set, so all child molecules will be labeled"
+                           " as 'good' regardless of quality. Please call Deriver.set_filter() first"
+                           " in order to use a filter for drug-likeness.")
+            filter_params = None
+
+        for _ in range(n_children):
+            parent_a, parent_b = np.random.choice(self.data.seed_smiles)
+            try:
+                new_child = co.crossover(parent_a, parent_b)
+                if new_child is not None:
+                    new_child = mu.mutate(new_child, mut_rate)
+                assert new_child
+                children.append(new_child)
+            except:
+                continue
```

Comment:
Way too broad (AssertionFailedError or something may work). Also at the very least needs to capture and log the exception before continuing.

Reply:
The exception handling I just added is the same as what is used in the original selfies deriver